easybuild/easyconfigs/s/SymEngine/SymEngine-0.11.2-gfbf-2023b.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'SymEngine'
+version = '0.11.2'
+
+homepage = 'https://github.com/symengine/symengine'
+description = "SymEngine is a standalone fast C++ symbolic manipulation library"
+
+toolchain = {'name': 'gfbf', 'version': '2023b'}
+
+source_urls = ['https://github.com/symengine/symengine/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['f6972acd6a65354f6414e69460d2e175729470632bdac05919bc2f7f32e48cbd']
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+]
+
+dependencies = [
+    ('GMP', '6.3.0'),
+    ('FLINT', '3.1.1'),
+    ('MPC', '1.3.1'),
+    ('LLVM', '16.0.6'),
+]
+
+local_opts = '-DWITH_OPENMP=ON -DWITH_SYMENGINE_RCP=ON -DWITH_COTIRE=OFF '
+local_opts += '-DWITH_MPFR=ON -DWITH_MPC=ON -DWITH_LLVM=ON -DWITH_BFD=ON -DWITH_FLINT=ON '
```

Comment:
Since MPFR is a direct dependency of this, it should be explicitly listed in the deps list.